exonum/tests/node.rs

```diff
@@ -18,22 +18,21 @@ extern crate futures;
 extern crate serde_json;
 extern crate tokio_timer;
 
-use futures::Future;
-use futures::sync::oneshot;
+use futures::{sync::oneshot, Future};
 use serde_json::Value;
 use tokio_timer::Timer;
 
-use std::sync::{Arc, Mutex};
-use std::thread::{self, JoinHandle};
-use std::time::Duration;
-
-use exonum::blockchain::{Service, ServiceContext, Transaction};
-use exonum::crypto::Hash;
-use exonum::encoding::Error as EncodingError;
-use exonum::helpers;
-use exonum::messages::RawTransaction;
-use exonum::node::{ApiSender, ExternalMessage, Node};
-use exonum::storage::{Database, Fork, MemoryDB, Snapshot};
+use std::{sync::{Arc, Mutex},
+          thread::{self, JoinHandle},
+          time::Duration};
+
+use exonum::{blockchain::{Service, ServiceContext, Transaction},
+             crypto::Hash,
+             encoding::Error as EncodingError,
+             helpers,
```

Comment:
I hate the alignment spaces, they are pain in the butt to work with if your IDE does not support it. But I am all for keeping rustfmt up to date.

Reply:
I'm not sure if this is `rustfmt`'s bug or not - I thought they must use block indent for everything now.

Reply:
I don't like such alignment either, but it didn't bring anything new comparing to the previous formatting:
```rust
use exonum::blockchain::...;
use exonum::crypto::...;

use exonum::{blockchain::...,
             crypto::...};
```

Reply:
I think that PR itself is okay and we must stick to default `rustfmt`'s style anyway, I just don't understand why visual indentation is used here - https://github.com/nrc/rfcs/blob/style-guide/style-guide/items.md#imports-use-statements - it should be a block indent :-\

Reply:
@ozkriff 
Will you create an issue for this problem? :upside_down_face:

Reply:
https://github.com/rust-lang-nursery/rustfmt/issues/2765